.travis.yml

```diff
@@ -7,6 +7,7 @@ cache:
     - $HOME/.cache/pip
 
 addons:
+  postgresql: "9.5"
```

Comment:
@luisg123v 9.6

Reply:
@ruiztulio 
Done. Also changed the target branch and edited the commit message to reflect the PostgreSQL version

Reply:
@luisg123v did you pushed the changes here? because I still see 9.5

Reply:
@ruiztulio 
You're right. I pushed to vauxoo-dev, as I use to do, but in this case it's a fork in my user.